TidepoolKit/TAPI.swift

```diff
@@ -387,8 +387,10 @@ public class TAPI {
         performRequest(request) { (result: DecodableResult<VerifyDeviceResponseBody>) -> Void in
             switch result {
             case .failure(let error):
+                TSharedLogging.debug("verify device request failed. deviceToken:\(deviceToken), error:\(error)")
```

Comment:
Just a quick [nit]: this will output the actual base64 `deviceToken`, right?  Otherwise we can't use it for testing later...